static/js/settings.js

```diff
@@ -94,9 +94,9 @@ function setup_settings_label() {
 
         // pm_mention_notification_settings
         enable_desktop_notifications: i18n.t("Visual desktop notifications"),
-        enable_offline_email_notifications: i18n.t("Email notifications when offline"),
-        enable_offline_push_notifications: i18n.t("Mobile notifications when offline"),
-        enable_online_push_notifications: i18n.t("Mobile notifications always (even when online)"),
+        enable_offline_email_notifications: i18n.t("Email notifications"),
+        enable_offline_push_notifications: i18n.t("Mobile notifications"),
+        enable_online_push_notifications: i18n.t("Mobile notifications for every message, including ones I read"),
```

Comment:
I don't think this is an accurate description of the behavior -- it doesn't matter whether you read the messages, what matters is whether you're active (e.g. we check whether you have a connected client and that your UserPresence is recent).

Reply:
I thought we send a push notification as long as the message is off screen?

Reply:
No -- the backend code that determines whether to send a push notification has no access to information about what view the user is looking at.  We send a **desktop notification** as long as the message is offscreen.  But the determination for whether to send a push/email notification is based on whether the user is likely to get a desktop notification, i.e. whether the user was present within the last few minutes on a client.